python-avocado.spec

```diff
@@ -441,11 +438,27 @@ examples of how to write tests on your own.
 %{_datadir}/avocado/yaml_to_mux_loader
 %{_datadir}/avocado/varianter_pict
 
+%package bash
```

Comment:
If I understand this correctly this moves the `libexec` files to a subpackage, right? I don't really like this idea as these are usually used from SimpleTests and people might get confused, because these are mentioned as Avocado features and not optional/additional parts.

Reply:
My goal is to:
 * reduce bloat (even if little) for the primary package
 * avoid content that is nowhere near useful to Python in a 'python-avocado' package
 * make the experience similar to the one with a pure Python module is installed (there is no libexec for shell in those cases)

Now, SimpleTests have a more broad concept, and what you mean is that SimpleTests *written in shell* can use those features.

I'll add a note to the docs:

http://avocado-framework.readthedocs.io/en/57.0/WritingTests.html#simple-tests-bash-extensions

To make it clearer that this feature may or may not be present (which is already a feature of `avocado exec-path`) and that when running out of packages, the `-bash` subpackage should be installed.

Having said all that, this is hardly a big issue, and we wouldn't be wrong in keeping it in the main package.  But it does look to me as a packaging improvement.

Reply:
Sure, I understand that. I'll checkout the new docs update but IMO the whole section should go to optional plugins and in writing tests should be only a link there...